### ISSUE TITLE
Sorting doesn't work with use_hubtraffic=True

### ISSUE THREAD

**Issue by mininator1** [EchterAlsFake/PHUB#49]
Hello nice library 👍 

I tried using the sorting, but it always returns the same videos. Can it be that it is broken?
I use it the follwoing way. Is that correct?

`query = client.search('query', Sort.VIDEO_MOST_RECENT, use_hubtraffic=True)`

A small question maybe also: Is there a page attribute and how can i set it? 
Keep it up 👍

**Comment by Egsagon** [comment]
Hey!
Sorting is a bit broken with hubtraffic, I don't know why but this should be an easy fix.
Instead for now, you can use regular web-scrapping by setting `use_hubtraffic=False`. This also allows more searching parameters than with hubtraffic.

There is no page attribute. Instead, you can do:
```py
page_index = 3
page = query._iter_page(query._get_page(page_index))
```
The reason why this is not a direct feature is because some queries have to provide pages with dynamic length. It's still something we don't know yet how to implement well.

Hope this helps, thanks for the support!

**Comment by Egsagon** [comment]
Should be fixed with v4.5. Params have been removed so you'll have to change your code. Use search by default but if you really want to use hubtraffic use client.search_hubtraffic.
Thanks for reporting
